lightningd/test/run-invoice-select-inchan.c

```diff
@@ -126,20 +126,20 @@ void json_add_address_internal(struct json_stream *response UNNEEDED,
 			       const char *fieldname UNNEEDED,
 			       const struct wireaddr_internal *addr UNNEEDED)
 { fprintf(stderr, "json_add_address_internal called!\n"); abort(); }
-/* Generated stub for json_add_amount_msat */
-void json_add_amount_msat(struct json_stream *result UNNEEDED,
-			  struct amount_msat msat UNNEEDED,
-			  const char *rawfieldname UNNEEDED,
-			  const char *msatfieldname)
-
-{ fprintf(stderr, "json_add_amount_msat called!\n"); abort(); }
-/* Generated stub for json_add_amount_sat */
-void json_add_amount_sat(struct json_stream *result UNNEEDED,
-			 struct amount_sat sat UNNEEDED,
-			 const char *rawfieldname UNNEEDED,
-			 const char *msatfieldname)
-
-{ fprintf(stderr, "json_add_amount_sat called!\n"); abort(); }
+/* Generated stub for json_add_amount_msat_compat */
+void json_add_amount_msat_compat(struct json_stream *result UNNEEDED,
+				 struct amount_msat msat UNNEEDED,
+				 const char *rawfieldname UNNEEDED,
+				 const char *msatfieldname)
+
+{ fprintf(stderr, "json_add_amount_msat_compat called!\n"); abort(); }
+/* Generated stub for json_add_amount_sat_compat */
+void json_add_amount_sat_compat(struct json_stream *result UNNEEDED,
+				struct amount_sat sat UNNEEDED,
+				const char *rawfieldname UNNEEDED,
+				const char *msatfieldname)
```

Comment:
Since you are already touching this line, rename `msatfieldname` to `satfieldname`

Reply:
No, these are always msat fields, since we unified on that as standard.